sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -55,6 +55,7 @@
     <Compile Include="Shared\DiagnosticScope.cs" />
     <Compile Include="Shared\DiagnosticScopeFactory.cs" />
     <Compile Include="Shared\DictionaryHeaders.cs" />
+    <Compile Include="Shared\EventSourceEvent.cs" />
```

Comment:
Since Core doesn't use this directly anywhere, do we need the reference here?

Reply:
Generally we don't compile sources that are not needed - It would show up in director view mode in VS, or all the time in VS Code 😄